tools/cxbe/Xbe.cpp

```diff
@@ -411,6 +405,17 @@ Xbe::Xbe(class Exe *x_Exe, const char *x_szTitle, bool x_bRetail, const std::vec
                     m_SectionHeader[v].dwSizeofRaw = RoundUp(r + 2, 4);
                 }
 
+                // calculate virtual size
+                if(v < m_Header.dwSections - 1)
+                    m_SectionHeader[v].dwVirtualSize =
+                        x_Exe->m_SectionHeader[v + 1].m_virtual_addr -
+                        x_Exe->m_SectionHeader[v].m_virtual_addr;
+                else
+                    m_SectionHeader[v].dwVirtualSize = MAX(
+                        RoundUp(x_Exe->m_SectionHeader[v].m_virtual_size, 4),
+                        m_SectionHeader[v].dwSizeofRaw
+                    );
```

Comment:
I always kind of assumed `raw size > virtual size` being fine because it's allowed in the PE format (`SizeOfRawData ` is rounded to `FileAlignment`, `VirtualSize` is exact). Does it cause an issue in the XBE loader?

Reply:
It appears so. ``XepLoadSection`` takes in an XBE section and, after loading, attempts to clear the tail part of the section.

**Retail 5838**
![image](https://github.com/XboxDev/nxdk/assets/1701315/b1fd3fee-0f7c-4453-a41c-777209ce14c4)